### DISCUSSION TITLE
Add View Mode

### DISCUSSION THREAD

**Discussion by lm93129** [rustdesk/rustdesk#2755] [General]
Added a view-only mode when remoting, which initially does not enable mouse and clipboard permissions, so that when you start remoting, it will not affect the running programs on the remote machine.

**Comment by fernandeusto** [comment]
<img width="391" alt="image" src="https://user-images.githubusercontent.com/87156183/224871289-b996aa99-af10-46b5-94c0-24d20932a249.png">

I already know the viewmode, but the idea is to connect in view mode to supervise the client and at a certain moment be able to switch to control mode without the intervention of the client to allow it, because it is already allowed in its client configuration. even when the connection management windows may be hidden

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/issues/3652 since this is proposed twice in one day, we decide to implement this. Please move there for discussion.
